assets/templates/rbac/prometheus-role-binding.yaml

```diff
@@ -2,7 +2,7 @@ kind: RoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: local-storage-metrics
-  namespace: openshift-local-storage
+  namespace: ${OBJECT_NAMESPACE}
```

Comment:
what if we just put this in the manifests directory and omitted this namespace line? let this be created wherever olm wills it (I assume it'll create it in the LSO namespace anyay)

Reply:
if that works, sure. How does it behave with upgrades? say I upgraded LSO from 4.8 (where namespace is `local-storage`) to 4.9 (in same namespace).

Reply:
worth trying.

Reply:
This actually works.  I removed the namespace in the prometheus-role-binding and prometheus-role .  Tried deploying via the catalog source using [this](https://github.com/openshift/local-storage-operator/blob/master/HACKING.md#for-developing-on-openshift-and-olm).

Both of these resources were deployed in the same namespace as csv.

Reply:
> How does it behave with upgrades?

I don't think olm behaviour on which namespace it places it in should change across upgrades